### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by snizhana-dynnyk**
towards https://github.com/giantswarm/giantswarm/issues/8695 How we use labels in GitHub Boards 

discussion notes - https://github.com/giantswarm/giantswarm/issues/8695#issuecomment-602442039

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/epic.md [hunk 1]
```diff
@@ -13,6 +13,17 @@ about: Template for our high-level story published in the external roadmap
 
 ---
 
+<!--
+Please assign appropriate labels:
+* product (for product area, choose between `product/KaaS`, `product/managed-services`, `product/empowerment`)
```
Comment (JosephSalisbury) [inline .github/ISSUE_TEMPLATE/epic.md]: do we wanna turn `product/KaaS` into `product/kaas`? that seems easier on my brain

Comment (snizhana-dynnyk) [reply .github/ISSUE_TEMPLATE/epic.md]: yes, Marian asked the same

Comment (snizhana-dynnyk) [inline .github/ISSUE_TEMPLATE/epic.md]:
```suggestion
* product (for product area, choose between `product/kaas`, `product/managed-services`, `product/empowerment`)
```

File: .github/ISSUE_TEMPLATE/epic.md [hunk 2]
```diff
@@ -13,6 +13,17 @@ about: Template for our high-level story published in the external roadmap
 
 ---
 
+<!--
+Please assign appropriate labels:
+* product (for product area, choose between `product/kaas`, `product/managed-services`, `product/empowerment`)
+* provider (for provider-specific features, choose between `provider/aws`, `provider/azure`, `provider/kvm`)
+* team (use team/null if you are not sure)
+*area (functional area if applicable, e.g. `area/observability` or `area/security`)
```
Comment (JosephSalisbury) [inline .github/ISSUE_TEMPLATE/epic.md]:
```suggestion
* area (functional area if applicable, e.g. `area/observability` or `area/security`)
```

**Pull request merged**